### ISSUE TITLE
enabling libretro should disable the UI

### ISSUE THREAD

**Issue by BPaden** [inolen/redream#104]
I think this is from imgui? I didn't see any option to actually disable it without manually editing cmakelists.txt, which does remove the top bar. I think it should be disabled by default for the libretro core.

**Comment by inolen** [comment]
Ah yes, I'll do that tonight.

This code:

```
# enable imgui
 target_link_libraries(redream ${IMGUI_LIBS})
 target_compile_definitions(redream PRIVATE ENABLE_IMGUI=1)
   
  # enable microprofile
  target_compile_definitions(redream PRIVATE ENABLE_MICROPROFILE=1)
 ```
needs to get moved into the above conditionals such that it only occurs for non-libretro builds.

**Comment by inolen** [comment]
Fixed with e06f450f029b77835e3948fa0e462e19c618daaf.
